### DISCUSSION TITLE
Can we set options for weightedBurgersVec calculations?

### DISCUSSION THREAD

**Discussion by JasonOttUW** [mtex-toolbox/mtex#2054] [Ask Anything] [answered]
Based on some recent reading on calculating the WBV from EBSD (in Wheeler et al., 2024; 'Using crystal-lattice distortion data for geological investigations: the weighted Burgers vector method', https://doi.org/10.1016/j.jsg.2023.105040), I am wondering if the weightedBurgersVec function as it is implemented allows for modification of the size of the cluster of points around each point where the WBV is being calculated by the differential method (e.g., the 'stencil' in the Wheeler et al., 2024 paper)?

**Comment by kilir** [comment, marked as answer]
Hi,

that option would be `'windowSize'` for both methods implemented in mtex. Beware, I think that the `'integral'` produces somewhat more reliable results as it depends less on the grid (at least for my testing ) and should be sufficiently fast in mtex.

Cheers,

Rüdiger

**Comment by JasonOttUW** [comment]
Hello, Rudiger, thanks for the swift response. I saw the windowSize in the options on the documentation page, but wasn't sure it applied to the differential method as well as the integral. For the windowSize, is it the distance from the center of the cluster or some other metric? (i.e., what would a window size of 3 look like relative to the point the calculation is made at? Thanks again for the help!
